library/src/main/java/ch/decent/sdk/api/MessagingApi.kt

```diff
@@ -0,0 +1,45 @@
+package ch.decent.sdk.api
+
+import ch.decent.sdk.DCoreApi
+import ch.decent.sdk.crypto.Credentials
+import ch.decent.sdk.model.*
+import ch.decent.sdk.net.model.request.GetMessageObjects
+import io.reactivex.Single
+import io.reactivex.functions.BiFunction
+
+class MessagingApi internal constructor(api: DCoreApi) : BaseApi(api) {
+
+  fun getAll(sender: ChainObject? = null, receiver: ChainObject? = null, maxCount: Int = 1000) =
+      GetMessageObjects(sender, receiver, maxCount).toRequest()
+
+  fun createMessageOperation(
+      credentials: Credentials,
+      to: ChainObject,
+      message: String
+  ) = Single.zip(
```

Comment:
pls explicitly specify return type for better readability of API method

Reply:
fixed